DXMainClient/DXGUI/Campaign/CampaignSelector.cs

```diff
@@ -309,7 +295,7 @@ private void BtnReturn_LeftClick(object sender, EventArgs e)
 
         private void BtnLaunch_LeftClick(object sender, EventArgs e)
         {
-            userSettings.ForEach(c => c.Save());
+            SaveSettings();
```

Comment:
`userSettings.ForEach(c => c.Save());` line was removed. 

`SaveSettings()` method is made for the campaign dropdowns and campaign checkboxes, which is different from `userSettings` here